components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/cdcPolling/InboundCDCConstants.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.inbound.endpoint.protocol.cdcPolling;
+
+class InboundCDCConstants {
+
+    /** Inbound Endpoint Parameters **/
+    public static final String CDC = "cdc";
+
+    //debezium
+    public static final String DEBEZIUM_NAME = "name";
+    public static final String DEBEZIUM_SNAPSHOT_MODE = "snapshot.mode";
+    public static final String DEBEZIUM_OFFSET_STORAGE = "offset.storage";
+    public static final String DEBEZIUM_OFFSET_STORAGE_FILE_FILENAME = "offset.storage.file.filename";
+    public static final String DEBEZIUM_OFFSET_FLUSH_INTERVAL_MS = "offset.flush.interval.ms";
+    public static final String DEBEZIUM_CONNECTOR_CLASS = "connector.class";
+    public static final String DEBEZIUM_DATABASE_HOSTNAME = "database.hostname";
+    public static final String DEBEZIUM_DATABASE_PORT = "database.port";
+    public static final String DEBEZIUM_DATABASE_USER = "database.user";
+    public static final String DEBEZIUM_DATABASE_PASSWORD = "database.password";
+    public static final String DEBEZIUM_DATABASE_DBNAME = "database.dbname";
+    public static final String DEBEZIUM_TABLES_INCLUDE_LIST = "table.include.list";
```

Comment:
Shall we name this `included.table.list` ?
Hope this means "list of tables to be included"

Reply:
it was discussed to allow the user to provide the parameters specified in debezium as it is in the synapse config xml file, to avoid confusions. Hence kept this set of properties as it is : https://debezium.io/documentation/reference/stable/connectors/mysql.html#_required_debezium_mysql_connector_configuration_properties